dame_flame/flame_algorithm.py

```diff
@@ -185,10 +194,20 @@ def flame_generic(df_all, treatment_column_name, outcome_column_name,
                   "No more covariate sets to consider dropping")
             break
                         
-        new_drop, pe, matched_rows, return_matches, bf = decide_drop(all_covs, 
+        new_drop, pe, matched_rows, return_matches, bf, units_in_g = decide_drop(all_covs, 
             consider_dropping, prev_dropped, df_all, treatment_column_name, 
             outcome_column_name, df_holdout_array, adaptive_weights, alpha, 
             df_unmatched, return_matches, C)
+# =============================================================================
```

Comment:
remove # = formatting and commented out code unless accompanied by a todo. Comment detailing the point of this code block